src/main/java/net/mcreator/element/types/Armor.java

```diff
@@ -163,6 +179,84 @@ public Model getBootsModel() {
 		return Model.getModelByParams(getModElement().getWorkspace(), bootsModelName, modelType);
 	}
 
+	public Model getHelmetItemModel() {
+		Model.Type modelType = Model.Type.BUILTIN;
+		if (helmetItemRenderType == 1)
+			modelType = Model.Type.JSON;
+		else if (helmetItemRenderType == 2)
+			modelType = Model.Type.OBJ;
+		return Model.getModelByParams(getModElement().getWorkspace(), helmetItemCustomModelName, modelType);
+	}
+
+	public Model getBodyItemModel() {
+		Model.Type modelType = Model.Type.BUILTIN;
+		if (bodyItemRenderType == 1)
+			modelType = Model.Type.JSON;
+		else if (bodyItemRenderType == 2)
+			modelType = Model.Type.OBJ;
+		return Model.getModelByParams(getModElement().getWorkspace(), bodyItemCustomModelName, modelType);
+	}
+
+	public Model getLeggingsItemModel() {
+		Model.Type modelType = Model.Type.BUILTIN;
+		if (leggingsItemRenderType == 1)
+			modelType = Model.Type.JSON;
+		else if (leggingsItemRenderType == 2)
+			modelType = Model.Type.OBJ;
+		return Model.getModelByParams(getModElement().getWorkspace(), leggingsItemCustomModelName, modelType);
+	}
+
+	public Model getBootsItemModel() {
+		Model.Type modelType = Model.Type.BUILTIN;
+		if (bootsItemRenderType == 1)
+			modelType = Model.Type.JSON;
+		else if (bootsItemRenderType == 2)
+			modelType = Model.Type.OBJ;
+		return Model.getModelByParams(getModElement().getWorkspace(), bootsItemCustomModelName, modelType);
+	}
+
+	public String getItemCustomModelNameFor(String part) {
+		return switch (part) {
+			case "helmet" -> helmetItemCustomModelName.split(":")[0];
+			case "body", "chestplate" -> bodyItemCustomModelName.split(":")[0];

```

Comment:
The `chestplate` case isn't needed

Reply:
I'm aware it's not used in our generators. However, I decided to add it in case someone makes an external generator and decides to use `chestplate` instead of `body` because he doesn't know our words/ways of working. I just wanted to avoid problems as everywhere in the software (UI) we use `chestplate` and not `body`. If you want it removed, I can remove it, but it's an easy problem-fixer.

Reply:
I think you should remove it. If someone sees their plugin doesn't generate proper code, they'd check this code to figure what value to use.

Reply:
Ok, it was just to make life a bit easier as this doesn't add a lot of code.